pynordpool/exceptions.py

```diff
@@ -19,5 +19,9 @@ class NordPoolResponseError(NordPoolError):
     """Response error from Nord Pool api."""
 
 
+class NordPoolEmptyResponseError(NordPoolError):
```

Comment:
Could have inherited from connection error. Then you would need to special case handle it _get. Just an idea.